src/sql/src/session/vars.rs

```diff
@@ -1257,26 +1257,35 @@ mod cluster_scheduling {
 ///
 /// Note that not all `ServerVar<bool>` are feature flags. Feature flags are for variables that:
 /// - Belong to `SystemVars`, _not_ `SessionVars`
-/// - Default to false and must be explicitly enabled
+/// - Default to false and must be explicitly enabled or provide an optional value and internal flag.
```

Comment:
I had trouble understanding this sentence without reading the actual code. Is it possible to add a doc-comment to each variant? Or maybe spell out all variants and their defaults here?

Reply:
Yep, updated the underlying PR. Thanks!